### ISSUE TITLE
Only Super Admin can mark quarantined mails as spam or ham

### ISSUE THREAD

**Issue by s256** [modoboa/modoboa-amavis#124]
Contrary to the docs only SuperAdmins can mark quarantined emails as spam or ham.
I set up two fresh servers (both with the official installer) on with postgres and the other with mysql.

To test another issue, but on both systems only the super admin can mark messages as spam or ham.

**Comment by tonioo** [comment]
@s256 Is it related to the other issue you opened: https://github.com/modoboa/modoboa-amavis/issues/123?

**Comment by s256** [comment]
@tonioo nope. Simple User and (at least) Domain Admins don't have the option to mark a mail as spam. The little drop down menu where you can "correct" amavis isn't there. I enabled Self-Service and direct release. 
This is not related to the other issue. Even if these buttons aren't doing anything, they still should be there. So this might just be a frontend issue. Triggering an actual learning is rather a backend issue I suppose. 

Let me know if I can test anything.

**Comment by tonioo** [comment]
@s256 Can you paste your settings here? (screenshot of the online parameters)

**Comment by s256** [comment]
Sorry, I can't. I disabled and removed the amavisd including the amavis plugin for modoboa. I'm using rspamd now. So I can only offer the django config.

@tonioo Can simple users and domain admins mark email as spam/ham in the webui on your instance?

**Comment by tonioo** [comment]
@s256 I've just tried on the demo site and it works but it is not enabled by default.
